### PR TITLE
Sandra plate map Ready for Stage

### PR DESCRIPTION
**Pull request by sandra444**
First version of the plate map maker (with pull from existing matrix) ready for testing on stage.

### CODE REVIEW

File: assays/ajax.py [hunk 1]
```diff
@@ -904,13 +905,13 @@ def get_data_points_for_charting(
     # Accommodation of "special" grouping
     group_sample_location = True
     group_method = True
-    group_time = True
+    grouse_time = True
```
Comment (Luke-Malchow-Bergenthal) [inline assays/ajax.py]: This variable isn't used right now, but I would prefer if its name remained "group_time"

File: assays/forms.py [hunk 2]
```diff
@@ -1587,10 +1595,10 @@ def clean(self):
         # test_file = None
 
         # TODO TODO TODO TODO TODO
-        if self.request and self.request.FILES and data.get('bulk_file'):
+        if self.request and self.request.FILES and data.get('plate_reader_file'):
```
Comment (Luke-Malchow-Bergenthal) [inline assays/forms.py]: It looks like you are editing the current upload form rather than modifying a copy of it, can you revert these changes? They will break study data uploads.

File: assays/templates/assays/assaystudy_index.html [hunk 3]
```diff
@@ -195,57 +197,4 @@ <h2>
         </div>
         {% endif %}
     </div>
-
-    {% if not user_is_group_admin and user_is_group_editor and not object.signed_off_by %}
```
Comment (Luke-Malchow-Bergenthal) [inline assays/templates/assays/assaystudy_index.html]: This is necessary for indicating that a study is ready for sign off, please do not delete it.

**Comment by sandra444** [comment]
I think it is fixed now.

**Comment by sandra444** [comment]
Thanks for pointing this out.
